keras-sprint/deeplabv3_plus.ipynb

```diff
@@ -0,0 +1,4573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install huggingface-hub
</pre><p>As a trick, you can add <code>%%capture</code> so the output is not displayed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/community-events/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I did remove it 😂😅